.github/dependabot.yml

```diff
@@ -18,3 +18,8 @@ updates:
       interval: "weekly"
     allow:
       - dependency-type: "direct"
+    ignore:
```

Comment:
Do we need to add `ts-node` here as well?

Reply:
@ullasholla looks like we don't really use `ts-node`. Scripts point to `test/index.ts` that do not exist https://github.com/SAP/open-ux-tools/search?q=ts-node

Is it ok to remove `test:local` scripts and `ts-node` dependency?

Reply:
The `examples` in the `axios-odata` branch use `ts-node`: https://github.com/SAP/open-ux-tools/blob/b34532e9c155a27e48639d3edf32152e37654324/examples/axios-odata/package.json#L9

Maybe keep `ts-node`? @tobiasqueck?

Reply:
I am ok to remove it and require to run `pnpm build` before running `pnpm start`. I mainly use it because I am lazy and  I don't want to run two scripts which I guess is not a valid reason to keep it ...

Reply:
I'm not fussed either way. Generally I'm a fan of lowering the barrier to explore and interact.